routes/index.js

```diff
@@ -0,0 +1,67 @@
+/*eslint no-undef: "error"*/
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
+/*eslint-env node*/
+
+const express = require('express');
+const router = express.Router();
+
+const alasql = require('alasql');
+require('../item_db.js');
+
+// Add your code here:
+router.param('itemId', (req, res, next, id) => {
+	const itemId = Number(id);
+	const item = alasql('SELECT * FROM items WHERE id='+ itemId);
+	const itemIndex = item.findIndex(item => item.id === itemId);
+	if ( isNaN(itemId)){
```

Comment:
you should probably do that check before you do findIndex

Reply:
Moved